packages/opentelemetry-resource-util/src/index.ts

```diff
@@ -17,8 +17,6 @@ import {Resource} from '@opentelemetry/resources';
 import {
   CLOUDPLATFORMVALUES_AWS_EC2,
   CLOUDPLATFORMVALUES_GCP_APP_ENGINE,
-  CLOUDPLATFORMVALUES_GCP_CLOUD_FUNCTIONS,
```

Comment:
Does this mean we are dropping support for Cloud Functions and Cloud Run in JS resource detector ?